firedrake/slope_limiter/vertex_based_limiter.py

```diff
@@ -0,0 +1,111 @@
+from firedrake import dx, assemble, LinearSolver
+from firedrake.function import Function
+from firedrake.functionspace import FunctionSpace
+from firedrake.parloops import par_loop, READ, RW
+from firedrake.ufl_expr import TrialFunction, TestFunction
+from limiter import Limiter
```

Comment:
Please use absolute imports (and `from __future__ import absolute_import` at top of file).